root/statistics/MusicBrainzHistory.js

```diff
@@ -18,7 +17,7 @@ component MusicBrainzHistory(events: $ReadOnlyArray<StatisticsEventT>) {
         ? events.map((event) => {
           const title = exp.l_statistics(
             '{date} - {title}',
-            {date: event.date, title: event.title},
+            {date: event.date, title: l_history(event.title)},
```

Comment:
As the translation is taking place in `all_events` already, the changes in this file seems to be unneeded.

Reply:
Oh, that's a fair point. Guess we still need `expand2react` then. Will revert.